### DISCUSSION TITLE
create new policy:

### DISCUSSION THREAD

**Discussion by FantZero** [eclipse-edc/Samples#285] [Q&A]
my Q: User A is the provider, user B is the consumer, A create asset1 based on student table, A create strategy policy1 (include log callback interface: http://localhost:19194/protocol/logback),  after B and A negotiate the contract, The log callback interface in policy1 is called by B when B get each row of data in asset1.
Who can provide the next create-policy.json and ConstraintFunction.java code in provider, thank you!

**Comment by harnvo** [comment]
Don't quite understand what in precise do you need to implement.

Why a `constraint function` should affect the behavior of `callback`? Is there anything you have already implemented to show us? And if so, what are the things you tried and what are the impediments?
